packages/back-end/src/middleware/authenticateApiRequestMiddleware.ts

```diff
@@ -137,10 +137,6 @@ export default function authenticateApiRequestMiddleware(
         project?: string | (string | undefined)[] | undefined,
         envs?: string[] | Set<string>
       ) => {
-        // Super admins have full access to every organization
-        if (req.user?.superAdmin) {
-          return;
-        }
```

Comment:
potential regression: it seems like superAdmins who are members no longer have super admin rights when accessing the API (instead, defaults to member permissions)

Reply:
I think it is fine to regress here. In general superAdmins probably already have admin role in the orgs they are part of.